src/components/User.tsx

```diff
@@ -13,34 +12,35 @@ const User = () => {
   switch (oidcUserLoadingState) {
     case OidcUserStatus.Loading:
       return (
-        <Button bordered disabled>
+        <div className="App-navbar__item App-navbar__item--loading">
           <UserIcon />
           Loading…
-        </Button>
+        </div>
       );
     case OidcUserStatus.Unauthenticated:
       return (
-        <Button
-          bordered
+        <button
+          className="App-navbar__item"
           onClick={() => { login('/').catch(logger.error); }}
+          type="button"
         >
           <UserIcon />
           Log in
-        </Button>
+        </button>
       );
```

Comment:
I'm afraid I don't quite follow why this one should be a lower-class `<button>` and not an upper-class `<Button>`?

Reply:
`button` is just a vanilla HTML element. `Button` is a custom component that renders a `button`, but adds the necessary classes for styling according to our design system, and adds properties for common variant states.

I avoided using a `Button` here because I didn't want any of its styling (and wasn't using any of its variant states). But arguably, for consistency, to support this case we should just add a boolean property like `<Button unstyled>` that forewent other styling. 🤔

I quite like that idea but if it's okay I'll hold that as an enhancement for later!

Reply:
That makes sense, thank you for the explanation! And yeah, definitely hold that as an enhancement for later.